odoo/fields.py

```diff
@@ -41,6 +42,12 @@
     'ir.model.relation', 'ir.model.constraint', 'ir.module.module',
 )
 
+PY2JS_REGEX_FLAGS = (
+    (re.I, "i"),
+    (re.M, "m"),
+    (re.S, "s"),
+)
```

Comment:
That's a mapping, I suggest to make it a `dict`.

Reply:
Would you care to explain why? What would be the advantage in this case?